pages/SPD/[id].js

```diff
@@ -19,12 +20,14 @@ export default function ServiceDetailsPage({ user }) {
   return (
     <>
       <ServiceCard id={id} user={user} />
-       <br />
-      <ActionLink href="/SPD">← Services</ActionLink>
-      <ActionLink href={`/SPD/services/${id}/edit`}>✏️ Edit</ActionLink>
-      <ActionButton type="danger" onClick={() => onDelete(id)}>
-        ❌ delete
-      </ActionButton>
+      <br />
+      <Flex style={{ justifyContent: "space-between" }}>
+        <ActionLink href="/SPD">← Services</ActionLink>
+        <ActionLink href={`/SPD/services/${id}/edit`}>✏️ Edit</ActionLink>
+        <ActionButton type="danger" onClick={() => onDelete(id)}>
```

Comment:
Please check if you use this property `type="danger"`. 
If not I would like to suggest to remove it. 🧹
```suggestion
        <ActionButton  onClick={() => onDelete(id)}>
```